src/test/config.sol

```diff
@@ -1669,5 +1669,65 @@ contract Config {
             calc_cut:     9900,
             lerp:         false
         });
+        afterSpell.collaterals["RWA008-A"] = CollateralValues({
+            aL_enabled:   false,
+            aL_line:      0,
+            aL_gap:       0,
+            aL_ttl:       0,
+            line:         30_000_000,
+            dust:         0,
+            pct:          5,
+            mat:          10_000,
```

Comment:
It is confusing putting the "_" separating the three zeros. In this case we are not talking about regular numbers but bps. So it is 100%. I'd just use "10000" like the rest of collaterals or case "100_00" (but we should change everything else which I wouldn't recommend for this PR).

Reply:
Fixed.

Reply:
resolving...